index.html

```diff
@@ -1512,113 +1499,146 @@ <h1>WoT Architecture</h1>
-                the device. A TD contains, among other things, a
-                device’s identifier, functions and attributes of a
-                device and communication protocols (i.e. transport
-                layer) information. More information about the TD can be
-                found in <a href="#sec-thing-description"></a>.
+                Linking does not only apply to hierarchical Things, but relations between Things and other resources in general.
+                Link relation types express how Things relate, for instance, a switch controlling a light or a room monitored by a motion sensor.
+                Other resources related to a Thing can be manuals, catalogs for spare parts, CAD files, a graphical UI, or any other document on the Web.
+                Overall, Web linking among Things makes the Web of Things browsable.
+                This can be further facilitated by providing Thing directories that manage an index of available Things, usually by caching their TD representation.
+                In summary, <a>WoT Thing Descriptions</a> MAY link to other <a>Things</a> and other resources on the Web to form a Web of Things.
             </p>
             <p class="ednote" title="TODO">
-                There is text incoming to explain the basics around Things, Consumers, and Intermediaries.
+                Figure to illustrate linked Things.
+            </p>
+            <p>
+                Things must be hosted on networked system components with a software stack to realize interaction through a network-facing interface, the <a>WoT Interface</a> of a <a>Thing</a>.
+                One example of this is an HTTP server running on an embedded device with sensors and actutors interfacing the physical entity behind the Thing abstraction.
+                However, W3C WoT does not mandate where <a>Things</a> are hosted; it can be on the IoT device directly, an <a>Edge device</a> such as a gateway box, or the cloud.
+            </p>
+            <p>
+                A typical deployment challenge is that local networks are not reachable from the Internet, usually because of IPv4 NATs or firewalls.
+                To remedy this situation, W3C WoT allows for <a>Intermediaries</a> between <a>Things</a> and <a>Consumers</a>.
+                <a>Intermediaries</a> can act as proxies for <a>Things</a>, where the <a>Intermediary</a> has a <a>WoT Thing Description</a> similar to the original <a>Thing</a>,
+                but which points to the <a>WoT Interface</a> provided by the <a>Intermediary</a>.
+                <a>Intermediaries</a> may also augment existing <a>Things</a> with additional capabilities or compose a new <a>Thing</a> out of multiple available <a>Things</a>,
+                thereby forming a virtual entity.
+                To <a>Consumers</a>, <a>Intermediaries</a> look like <a>Things</a>, as they possess <a>WoT Thing Descriptions</a> and provide a <a>WoT Interface</a>,
```

Comment:
perhaps "look and behave like things", but i would argue that they are just things with the additional virtualisation aspect.

Reply:
Then what is this virtualization aspect? How would you define it?